css/New Evaluation.css

```diff
@@ -0,0 +1,46 @@
+body{
```

Comment:
I would very much avoid giving files similar names, uness the files are covering different concerns (ie html vs css vs js - it makes sense to have intuitive and similar naming conventions).

Even though those files are duplicates I wanted to point out the following:
In this case you have 2 files `New Evaluation.css` and `NewEvaluation.css`. It is very easy for a colleague to start working on the wrong file due to a naming confusion.